src/SquirrelInstaller/Bloom.nuspec

```diff
@@ -13,7 +13,10 @@
 	<iconUrl>https://s3.amazonaws.com/bloomlibrary.org/squirrel/BloomSetup.ico</iconUrl>
   </metadata>
 	<files>
-		<file src="..\..\output\release\**\*.*" exclude="*.xml;Bloom.vshost.exe;AWSSDK.pdb;remoteDebugging" target="lib\net45\"/>
+		<file src="..\..\output\release\**\*.*" exclude="*.xml;Bloom.vshost.exe;**\*.pdb;**\BloomTests.*;remoteDebugging" target="lib\net45\"/>
+		<!-- Having excluded all pdbs, we do want the main one for the Bloom exe, so we get better stack dumps.
+		This pattern should cover Bloom.pdb or BloomAlpha.pdb or anything similar (but we don't need BloomTests.pdb or BloomChorusPlugin.pdb).-->
```

Comment:
We have not been shipping them in the past. Since we didn't do anything obvious to suppress them, I suspect we are not fetching them in the first place when we get dependencies. So we'd have to change more things (and make the installer larger again) to start getting them. It might be worthwhile, but I don't see it as part of this task. @hatton, what do you think? I don't recall missing libpalaso stack details while dealing with Bloom crashes.